### PR TITLE
Update `AppleAccelerate` for 1.0

### PR DESCRIPTION
**Pull request by staticfloat**
Builds on initial PR by `@platawiec`.

### CODE REVIEW

File: src/Util.jl [hunk 1]
```diff
@@ -6,34 +6,39 @@ tupletypelength(a)=length(a.parameters)
 macro replaceBase(fs...)
```
Comment (simonbyrne) [inline src/Util.jl]: we should probably just get rid of this, since it doesn't make sense anymore.

Comment (staticfloat) [reply src/Util.jl]: Why doesn't it make sense anymore?  I just tested it out and it seems to work alright.

**Comment by simonbyrne** [comment]
May need to update the .travis.yml file as well.

File: src/Util.jl (hunk 1 above)
Comment (simonbyrne) [reply src/Util.jl]: we don't define `log(x::AbstractArray)`

Comment (simonbyrne) [reply src/Util.jl]: or if we do (e.g. `exp(x::Matrix)`) it means something else

Comment (staticfloat) [reply src/Util.jl]: Part of this update is to override the broadcasting machinery such that `log.(x)` calls `log(x)`; I'm not sure what the right way to do that while still replacing `base` is though.

Comment (simonbyrne) [reply src/Util.jl]: One option would be to give them `v` prefixes, i.e.`vadd`, `vlog`, etc. by default. Then we could have another function which would do the type piracy for us.

**Pull request merged**